lib/decorators/index.ts

```diff
@@ -1,12 +1,30 @@
+/**
+ * 2021-04-07 by MrSpoocy: Fix issue if you use ES6 or higher (https://github.com/vuex-orm/vuex-orm/commit/25b458bd4de72f64ad4e8b3a3983ace062be8cfd)
```

Comment:
Can we remove these comments? We prefer only having Github handles in the commit history and not in the source code. Also, since the arguments are not documented this function documentation does not add much value.

Reply:
Yes you can remove it :)